config/routes.rb

```diff
@@ -57,15 +57,8 @@
     end
   end
 
-  get "/:year/:month/:day/:slug" => "blog#post",  :year   => /\d{4}/,
```

Comment:
this is going to return 404 for external links to these pages no?

Reply:
Yes, though I didn't think we ever actually used that form for our links. However, digging in the history, it seems that we switched as recently as 2013. That's a long time ago, but the blog posts are older than that. So it's probably worth leaving a redirect.

Reply:
nice :)